src/app/services/ActivesPartsService.ts

```diff
@@ -1,84 +1,83 @@
 import { Injectable, OnDestroy } from '@angular/core';
 import { BehaviorSubject, Observable } from 'rxjs';
 import { PartDAO } from '../dao/PartDAO';
-import { ICurrentPartId, IPart } from '../domain/icurrentpart';
+import { IPart, IPartId } from '../domain/icurrentpart';
 import { FirebaseCollectionObserver } from '../dao/FirebaseCollectionObserver';
-import { assert } from '../utils/utils';
+import { assert, Utils } from '../utils/utils';
+import { MGPOptional } from '../utils/MGPOptional';
 
 @Injectable({
     providedIn: 'root',
 })
+/*
+ * This service handles parts being played, and is used by the server component and game component.
```

Comment:
not part being played (because the user don't play them when seeing them, and because some are waiting joiner) so "this service handles active parts"

Reply:
description mise à jour